diff --git a/component/wifi/driver/src/core/option/rtw_opt_skbuf_rtl8735b.c b/component/wifi/driver/src/core/option/rtw_opt_skbuf_rtl8735b.c
index f881a36..e88df27 100644
--- a/component/wifi/driver/src/core/option/rtw_opt_skbuf_rtl8735b.c
+++ b/component/wifi/driver/src/core/option/rtw_opt_skbuf_rtl8735b.c
@@ -4,7 +4,7 @@
 #include <skbuff.h>
 
 #define MAX_SKB_BUF_SIZE     (1792 + (4000-1514))	// should >= the size in wlan driver
-#define MAX_SKB_BUF_NUM      512
+#define MAX_SKB_BUF_NUM      1024
 #define MAX_LOCAL_SKB_NUM    (MAX_SKB_BUF_NUM + 2)
 
 /* DO NOT modify skb_buf and skb_data structure */
