diff --git a/lib/wslay_net.h b/lib/wslay_net.h
index 2310870..b271cff 100644
--- a/lib/wslay_net.h
+++ b/lib/wslay_net.h
@@ -34,6 +34,13 @@
 #ifdef HAVE_ARPA_INET_H
 #  include <arpa/inet.h>
 #endif /* HAVE_ARPA_INET_H */
+
+#include <lwip/def.h>
+#define htons(x) lwip_htons(x)
+#define ntohs(x) lwip_ntohs(x)
+#define htonl(x) lwip_htonl(x)
+#define ntohl(x) lwip_ntohl(x)
+
 #ifdef HAVE_NETINET_IN_H
 #  include <netinet/in.h>
 #endif /* HAVE_NETINET_IN_H */
